flutter_news_example/lib/app/router/app_router.dart

```diff
@@ -33,8 +40,15 @@ class AppRouter {
     GoRoute(
       path: LoginPage.path,
       name: LoginPage.path,
-      pageBuilder: (context, state) => const MaterialPage<void>(
-        child: LoginPage(),
+      builder: (context, state) => SignInScreen(
```

Comment:
And last question, shouldn't this be wrapped in a `MaterialPage` like the other pages?

Reply:
I think we should avoid pageBuilder unless we want a custom animation in the navigation, MaterialPage is the default with builder and MaterialApp.

ref:

```
/// The widget returned by [builder] is wrapped in [Page] and provided to the
/// root Navigator, the nearest ShellRoute ancestor's Navigator, or the
/// Navigator with a matching [parentNavigatorKey].
///
/// The Page depends on the application type: [MaterialPage] for
/// [MaterialApp], [CupertinoPage] for [CupertinoApp], or
/// [NoTransitionPage] for [WidgetsApp].
```

Reply:
Perfect, in that case I'll create another PR removing it 👍